README.md

```diff
@@ -62,12 +64,14 @@ List of GAME ID's in Uplay by Ubisoft
 
 # Watch_Dogs Franchise
 274 - Watch_Dogs  
+541 - Watch_Dogs (Steam)  
```

Comment:
This makes 2 IDs for `Watch_Dogs (steam)`. Not sure if 741 is a typo, or different version.